packages/ag-charts-community/src/chart/legend/legend.ts

```diff
@@ -516,26 +513,24 @@ export class Legend extends BaseProperties {
             paddedSymbolWidth += spacing + markerWidth;
         }
 
-        if (this._symbolsDirty) {
-            const { marker, line } = markerLabel;
+        const { marker, line } = markerLabel;
```

Comment:
Verified this change has no impact on performance - it doesn't re-render the marker/line if they didn't change